build.gradle

```diff
@@ -68,9 +71,11 @@ dependencies {
     // switch to implementation for debugging
     modCompileOnly 'maven.modrinth:starlight:1.1.2+1.20'
 
-    modCompileOnly 'maven.modrinth:sodium:mc1.20.1-0.5.3'
+    modCompileOnly ':sodium-fabric-0.6.0-snapshot+mc1.20.1-local'
     modCompileOnly 'maven.modrinth:iris:1.6.10+1.20.1'
 
+    modCompileOnly 'maven.modrinth:embeddium:kJkE9dfS'
```

Comment:
Use the version number (`0.3.25+mc1.20.1`) instead of the version ID and move this line to be right after the Sodium dependency without additional newlines.

Reply:
Using a version number will pull the forge version since they have the same version numbers